src/SCTPWrapper.cpp

```diff
@@ -98,6 +98,42 @@ void SCTPWrapper::OnNotification(union sctp_notification *notify, size_t len) {
       break;
     case SCTP_STREAM_RESET_EVENT:
       SPDLOG_TRACE(logger, "OnNotification(type=SCTP_STREAM_RESET_EVENT)");
+      struct sctp_stream_reset_event reset_event;
+      reset_event = notify->sn_strreset_event;
+      for (int i = 1; i < 2; i++) {
```

Comment:
The event could hold multiple streams as it's a [list](https://tools.ietf.org/html/rfc6525#section-6.1.1). I would ideally want to loop across each of them, but I'm not sure how to do that without knowing the size.